tests/meson.build

```diff
@@ -0,0 +1,85 @@
+ma_test_broken = disabler()
+
+test(
+  'automated',
+  executable(
+    'ma_test_automated',
+    'test_automated/ma_test_automated.c',
+    dependencies: ma_sys_deps,
+    install: false,
```

Comment:
May also want `build_by_default: false`, which means these test programs won't be built when running `ninja all` but will be built when running `ninja test` (or `meson test`).

(Personally I would then take the opportunity to remove the option itself, as it is then no longer needed for the purpose of avoiding useless compilations when just building and installing.)